developer-docs-site/docs/guides/nfts/mint-nft-cli.md

```diff
@@ -4,587 +4,291 @@ title: "Mint NFTs with the Aptos CLI"
+This tutorial is intended to demonstrate how to programmatically mint NFTs on Aptos. It builds upon the simplest version of an NFT Collection creator manually minting an NFT to a receiver's account to eventually creating a more complex smart contract that facilitates users requesting an NFT and receiving it without ever having to get explicit signer approval from the Collection creator.
 
 ## Prerequisites
 
 This tutorial assumes you have:
 
-* a [GitHub account](https://docs.github.com/en/get-started/signing-up-for-github/signing-up-for-a-new-github-account)
-* the [GitHub CLI](https://cli.github.com/)
 * the [Aptos CLI](../../tools/install-cli/index.md) (or you can run from [aptos-core](https://github.com/aptos-labs/aptos-core) source via `cargo run`)
 * the `aptos-core` repository checked out: `git clone https://github.com/aptos-labs/aptos-core.git`
+* a general understanding of NFTs and NFT Collections
+* an understanding of [the Token V2 standard](https://aptos.dev/guides/nfts/token-v2)
 
-Then:
+## 1. Create a simple smart contract to mint an NFT
 
-* Review the [mint_nft](https://github.com/aptos-labs/aptos-core/tree/main/aptos-move/move-examples/mint_nft) source code and code comments within each subdirectory.
-* Explore the `mint_event_ticket` function defined within each subdirectory.
+We're going to start by making the simplest form of the flow for creating a collection and minting a token and sending it to a user. The code for this part of the tutorial is in the first section of the `move-examples/mint_nft` folder in the aptos-core repository.
 
-## NFT types
+Here are the most basic things we need to do first:
 
-The two most common types of NFT are event ticket / certificates and profile picture NFTs.
+* Create the NFT collection and store the configuration options for it
+* Mint a non-fungible token within that collection using the configuration options
+* Send the minted token to a user account
 
-### Event tickets and certificates
+### Defining configuration options
 
-This kind of NFT has a base token, and every new NFT generated from this base token has the same token data ID and image. They are generally used as certificates, meaning each NFT created from the base token is considered a printing edition of the base token.
+The first thing we need to do is store the collection information somewhere. These will be fields we get on-chain later to mint the token.
 
-You might use this type of NFT for event tickets where each NFT is a ticket and has properties representing expiration date and if the ticket has been used. When you mint the NFT, you may set an expiration time for the event ticket and `is_ticket_used` to `false`. When the ticket is used, you update `is_ticket_used` to `true`.
+We'll store it in a `struct` called `MintConfiguration`. These fields aren't the only fields used to create a collection, but they are the most basic ones meant to demonstrate how to store configuration details.
```

Comment:
This sentence can be merged into the previous sentence to say something along the line of:

We need to store collection information so we can mint tokens later such as:
[MintConfiguration struct code with annotated comments explaining the fields]